source/testing/testing-components.md

```diff
@@ -196,38 +245,39 @@ of a test double (dummy function).
 external action is called:
 
 ```tests/integration/components/comment-form-test.js
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
-import { run } from '@ember/runloop';
 
-moduleForComponent('comment-form', 'Integration | Component | comment form', {
-  integration: true
-});
+module('Integration | Component | comment form', function(hooks) {
+  setupRenderingTest(hooks);
 
-test('should trigger external action on form submit', function(assert) {
-  assert.expect(1);
+  test('should trigger external action on form submit', async function(assert) {
+    assert.expect(1);
 
-  // test double for the external action
-  this.set('externalAction', (actual) => {
-    let expected = { comment: 'You are not a wizard!' };
-    assert.deepEqual(actual, expected, 'submitted value is passed to external action');
-  });
+    // test double for the external action
+    this.set('externalAction', (actual) => {
+      let expected = { comment: 'You are not a wizard!' };
+      assert.deepEqual(actual, expected, 'submitted value is passed to external action');
+    });
 
-  this.render(hbs`{{comment-form submitComment=(action externalAction)}}`);
+    await render(hbs`{{comment-form submitComment=(action externalAction)}}`);
 
-  // fill out the form and force an onchange
-  this.$('textarea').val('You are not a wizard!');
-  this.$('textarea').change();
+    // fill out the form and force an onchange
+    this.element.querySelector('textarea').value = 'You are not a wizard!';
+    this.element.querySelector('textarea').dispatchEvent(new Event('change'));
```

Comment:
this should use `await fillIn('textarea', 'You are not a wizard!');`

Reply:
same as above